core/server/api/v2/invites.js

```diff
@@ -6,7 +6,7 @@ const urlService = require('../../services/url');
 const settingsCache = require('../../services/settings/cache');
 const models = require('../../models');
 const api = require('./index');
-const ALLOWED_INCLUDES = ['created_by', 'updated_by'];
+const ALLOWED_INCLUDES = [];
```

Comment:
Do we still need this property if it's empty?

Reply:
If we add an activity stream (actions), we need to add new includes. So i would just leave it as it is. Doesn't hurt 😎